packages/next/src/build/index.ts

```diff
@@ -942,7 +941,7 @@ export default async function build(
                 ]
               : []),
           ]
-            .filter(nonNullable)
+            .filter((item): item is NonNullable<string> => item != null)
```

Comment:
How about changing the existing function in [`non-nullable.ts`](https://github.com/vercel/next.js/blob/canary/packages/next/src/lib/non-nullable.ts) instead?